docs/zh-CN/sql-reference/sql-statements/Data Definition/CREATE TABLE.md

```diff
@@ -66,7 +66,7 @@ under the License.
             范围：-2^127 + 1 ~ 2^127 - 1
         FLOAT（4字节）
             支持科学计数法
-        DOUBLE（12字节）
+        DOUBLE（8字节）
```

Comment:
Please change the title of PR in English.
Also the en/sql-relerence/sql-statements/Data Defination/CREATE TABLE.md shoudl be changed.

Reply:
Done